core/service/json_parse.hpp

```diff
@@ -14,52 +14,57 @@ namespace json_parse {
         INT,
         FLOAT,
         LIST,
-        DICT
+        DICT,
+        INVELIED
     };
 
     struct Object {
-        Type type;
+        const Type type;
         int integer;
         std::string str;
         bool boolean;
         float floating;
 
-        Object(Type t,int i) :
-            type(t), integer(i)
+        std::map<std::string, Object> dictSub;
+        std::vector<Object> listSub;
+
+        Object(const Type& t,const int& i) :
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Member variable 'Object::boolean' is not initialized in the constructor.](https://www.codacy.com/app/soramitsu/iroha/file/4478583033/issues/source?bid=3712491&fileBranchId=3923843#l31)